src/Transport.ts

```diff
@@ -107,6 +117,18 @@ export class Transport {
                     this._channel.send(msg)
                 })
             })
+
+            // Also send the list of events this end is interested in so the far
+            // end can know when to wait for this end to be ready when triggering
+            // a specific slot, and when NOT to wait.
+            // This is necessary only when some events have been blacklisted
```

Comment:
We should add documentation about the blacklist and how to use it since it's a newly introduced option. Also since we're introducing it we might want to go with `disallowList`/`allowList` naming :)

Reply:
I have renamed it "ignoredEvents"